activiti-engine/src/main/resources/org/activiti/db/create/activiti.postgres.create.engine.sql

```diff
@@ -205,6 +209,7 @@ create table ACT_RU_TASK (
     TENANT_ID_ varchar(255) default '',
     FORM_KEY_ varchar(255),
     CLAIM_TIME_ timestamp,
+    APP_VERSION_ integer,
```

Comment:
IMO, as far as we keep other database scripts here, we need to keep them up to date. Could you, please, update the script for other DBs?

Reply:
Alright, I can add it to others as well, yes